spec/ddtrace/contrib/action_cable/patcher_spec.rb

```diff
@@ -0,0 +1,93 @@
+require 'spec_helper'
+require 'ddtrace/contrib/analytics_examples'
+
+require 'rails'
+require 'active_support'
+require 'ddtrace'
+RSpec.describe 'ActionCable patcher' do
+  let(:tracer) { get_test_tracer }
+  let(:configuration_options) { { tracer: tracer } }
+
+  def all_spans
+    tracer.writer.spans(:keep)
+  end
+
+  before(:each) do
+    if Datadog::Contrib::ActionCable::Integration.compatible?
+      Datadog.configure do |c|
+        c.use :action_cable, configuration_options
+      end
+    else
+      skip
+    end
+  end
+
+  around do |example|
```

Comment:
Out of scope for this PR, but we should consider migrating this kind of behavior into span test helpers like you had started with error matchers.

Reply:
Funny you say this, I did this exact thing in my PR where I'm migrating Rails tests form Minitest to RSpec. It helps legibility quite a lot!